configure.ac

```diff
@@ -447,6 +449,14 @@ if test "x$have_cryptopp" = "xyes"; then
     #AC_SUBST([CRYPTO_CXXFLAGS], [$CRYPTOPP_CXXFLAGS])
     AM_CXXFLAGS="${AM_CXXFLAGS} ${CRYPTOPP_CXXFLAGS}"
     AC_SUBST([CRYPTO_LIBS], [$CRYPTOPP_LIBS])
+    # OpenSSL -- defines OPENSSL_{INCLUDES,LIBS,LDFLAGS} on success
+    AX_CHECK_OPENSSL([], [found_openssl=1], [found_openssl=0])
```

Comment:
no, it's not cosmetic. the first arg is executed when openssl is found, and the second arg is executed when openssl is not found. in your case, the second arg is `[found_openssl=1]`.